jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -273,13 +264,9 @@ public void afterTaskFailedEvent(TaskEvent event) {
 
     @Override
     public void afterTaskAddedEvent(TaskEvent event) {
-        String userId = event.getTaskContext().getUserId();
         Task ti = event.getTask();
         TaskPersistenceContext persistenceContext = getPersistenceContext(((TaskContext)event.getTaskContext()).getPersistenceContext());
         try {
-            if(ti.getTaskData().getProcessId() != null){
-                userId = ti.getTaskData().getProcessId();
-            }
```

Comment:
Failing tests due to this removal, I think it's a fix (though result changes) because here it's overwriting the userId from the context.